Week 2/Weekly Practice Problems (Week 2)/1688: Count of Matches in Tournament/agunn.java

```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public int numberOfMatches(int n) {
+        int count = 0;
+        while (n > 0) {
+            // Add matches to count
+            count = count + n/2;
```

Comment:
`count += n/2` could also work for this statement, but I'm just showing the option since this is a personal preference thing.